catalog-info.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
why do we have this and `template/model-server/catalog-info.yaml` ?

if there is difference in content between the 2 files I am missing it.

Reply:
Oops, that wasn't supposed to be committed. It's been removed now.